repositories/ResumeRepository.ts

```diff
@@ -1,8 +1,22 @@
-import { ResumeModel } from 'models/ResumeModel';
 import { EntityRepository } from 'typeorm';
+import { ResumeModel } from '../models/ResumeModel';
 import { BaseRepository } from './BaseRepository';
+import { Uuid } from '../types';
+import { UserModel } from '../models/UserModel';
 
 @EntityRepository(ResumeModel)
 export class ResumeRepository extends BaseRepository<ResumeModel> {
+  public async findVisibleResumes(): Promise<ResumeModel[]> {
+    return this.repository.find({
+      isResumeVisible: true,
```

Comment:
Should we filter duplicates from one person?

Reply:
At some point we may support users having multiple resumes, and we also have the guarantee that there'll only be one resume per-person right now due to our resume deletion logic, so I think it's okay to not do any duplicate filtering here (also to keep API types consistent so that we wouldn't have to switch from returning Resume to returning Resume[] if we do need to support multiple resumes.